src/node_crypto.cc

```diff
@@ -751,6 +751,23 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
   CHECK_EQ(sc->ca_store_, nullptr);
 
   if (!root_cert_store) {
+#if defined(NODE_SYSTEM_CERT_STORE)
+    // *Assume* OpenSSL is setup correctly, which is the case
+    // for distribution supplied versions.
```

Comment:
@bnoordhuis I agree with @AdamMajer here, node's copy of OpenSSL uses a cert path in `/etc`, not `/usr/local`